### ISSUE TITLE
Mark 16 - Footnotes

### ISSUE THREAD

**Issue by ChrisJarka** [Door43/udb-en#9]
After seeing how the footnotes in this chapter appear in the tS-Android app I wonder if the Source Text could rethink the information in the footnotes?  
IMHO - The footnotes that refer to what was said in the footnote on verse 9 are not helpful as the verse 9 chunk gets further away from the chunk the translator is looking/working on at that time.

**Comment by TomWarren** [comment]
I can take a look in the morning. We are 10 hours ahead in Nepal, and it will be your evening before I get to it. The footnotes are tricky, so I will make it. I will look at the Android Trans Studio app.

**Comment by TomWarren** [comment]
I looked at this section. Most modern translations either consign this longer ending of Mark to a long footnote, or they include it in an addendum after the end of Mark. The section is very dubious and is considered a late addition, by someone other than Mark (the vocabulary and grammar are very much un-Markan. I noticed that the Android has a little book (indicating a footnote), but the Desktop app just reads the content of the footnote. In either case, it is important to separate the longer ending of Mark from the inspired text, because of the late date, the lack of continuity with Mark, the positioning of this pericope in other Gospels, other than Mark, and the weak and late textual support for including it. It is included, it would seem, because those familiar with the KJV would feel that the text was somehow abridged or tampered with, when the opposite is true. We are trying to be faithful to the text, to the best text.

**Comment by pjoakes** [comment]
Tom,

We also need to include it for those translators who are following the respected text in their area and want it in their translation. Our footnote needs to allow for that.

Perry

Perry Oakes, Ph.D.
Chief Language Officer
perry_oakes@wycliffeassociates.org<mailto:perry_oakes@wycliffeassociates.org>

[cid:image002.png@01D116FB.66313BC0]

On May 11, 2016, at 6:20 AM, Tom Warren <notifications@github.com<mailto:notifications@github.com>> wrote:


I looked at this section. Most modern translations either consign this longer ending of Mark to a long footnote, or they include it in an addendum after the end of Mark. The section is very dubious and is considered a late addition, by someone other than Mark (the vocabulary and grammar are very much un-Markan. I noticed that the Android has a little book (indicating a footnote), but the Desktop app just reads the content of the footnote. In either case, it is important to separate the longer ending of Mark from the inspired text, because of the late date, the lack of continuity with Mark, the positioning of this pericope in other Gospels, other than Mark, and the weak and late textual support for including it. It is included, it would seem, because those familiar with the KJV would feel that the text was somehow abridged or tampered with, when the opposite is true. We are trying to be faithful to the text, to the best text.

**Comment by ChrisJarka** [comment]
I'm not part of the debate of having a note or what to explain - my concern is - "Are the sequential notes end user friendly?"
When the end-user is in tS-Android and clicks on the footnote icon in the chunk for Mark 16:9-11 they see the comment: "The best ancient copies omit Mark 16:9-20."
When the end-user is in tS-Android and clicks on the footnote icon in the chunk for Mark 16:12-13; they see the comment: "See note at verse 9." In which, if they have forgotten what the note said for verse 9 requires them to close the footnote screen, swipe to the 9-11 chunk and then open the footnote to read it again.
And when the end-user is in tS-Android and clicks on the footnote icon in the chunk for Mark 16:14-16; they see the comment: "See note at verse 9." In which, if they have forgotten what the note said for verse 9 requires them to close the footnote screen, swipe to the 9-11 chunk and then open the footnote to read it again.
Now, hopefully they have memorized and will not forget what the note on chunk 9-11 states - but if they do forget, then for the chunks of 17-18 and 19-20 they will need to close the footnote screen, swipe to the 9-11 chunk and then open the footnote to read it again.

So, does the text team think this formatting is end-user friendly?
By the way - in my inexpert opinion - I think the note on chunk 9-11 is good.

**Comment by TomWarren** [comment]
Because of the chunks each "included" text must be identified as related to the section as a whole which is not supported by the best manuscripts. If we list the verses without the reference to VS 9, the reader may conclude that the chunk is approved, when it is not.

**Comment by pjoakes** [comment]
That sounds like a workable solution.

Perry Oakes, Ph.D.
Chief Language Officer
perry_oakes@wycliffeassociates.org<mailto:perry_oakes@wycliffeassociates.org>

[cid:image002.png@01D116FB.66313BC0]

On May 11, 2016, at 10:56 AM, Tom Warren <notifications@github.com<mailto:notifications@github.com>> wrote:


Because of the chunks each "included" text must be identified as related to the section as a whole which is not supported by the best manuscripts. If we list the verses without the reference to VS 9, the reader may conclude that the chunk is approved, when it is not.

**Comment by SusanQuigley** [comment]
When end-users read the chunks for Mark 16:12, 14, 17, and 19, the will see the following footnote: "See footnote at verse 9." 
If they are using an android, it will take them extra effort to go back to the chunk for Mark 16:9 so they can go back to see what it says and then get back to the chunk they were reading. Instead of having end-users go back to the v9 chunk, could each one of those chunks have the following note? "The best ancient copies omit  Mark 16:9-20."

**Comment by TomWarren** [comment]
Ok. Let's try that.

**Comment by SusanQuigley** [comment]
Tom, thanks for being willing to try that. But after talking with Ed, I realize that is probably not the best solution. We have encouraged translators to translate the ULB footnotes for their readers. If the translators see a footnote in each chunk of  Mark 16, they may think that they are expected to translate it each time.  But their readers won't need this if they use the unfoldingWord app, because the uW app displays each chapter, not each chunk. So the end-readers will only need to see it in Mark 16:9.  
If the translators need to be reminded in each of the rest of the chunks that Mark 16:9-20 is not in the best manuscripts, maybe there should be a translation note about it in each of those chunks rather than a footnote.

**Comment by TomWarren** [comment]
The difficulty is that we cannot accommodate to every way in which the text will be presented. One of the limitations is that, because of these limitations, there will be repetition at times. I think the long ending of Mark is one of only a few situations where we just tolerate less than ideal form for the sake of protecting the reader from assuming that the text is not supported and should be separated from the inspired text. Perhaps we should leave it like this (with the reminder at each chunk), or, if that doesn't work, we should set the longer section apart with brackets (as you suggested earlier) or add the text at the end as a stand-alone appendix. I vote we keep it as is, with the reminder, with the comfort that the MTT will be reminded many times that this text is not supported by the best copies.

**Comment by SusanQuigley** [comment]
OK.

**Comment by ChrisJarka** [comment]
Tom - thank you for understanding my point of view and making the changes. Let's hope that they will show up correctly in all programs.

**Comment by TomWarren** [comment]
All is well.

**Comment by ChrisJarka** [comment]
The programmers/developers of tS are working on the displaying of footnotes. As the API is updated and the footnote display is being developed for tS-Desktop and uW we will not know the result of the placement of footnotes until both updates are achieved. Expected goal of tS-D version 9 and API 3. Assigned to Sprint 5.16 https://github.com/unfoldingWord-dev/ts-desktop/issues/365

**Comment by TomWarren** [comment]
There may be a need to identify major issues in the Greek and Hebrew (and Aramaic) texts. Suggesting using Aland/ Metzger A Textual Commentary, not to copy, but to see where "D" readings are.  This would be a pretty large project to identify the variant readings and then to put up footnotes for them. The OT textual issues are much narrower, but we are adding them along the way. Just to put that up. I have some OT textual criticism texts.

**Comment by pohlig** [comment]
In general, I had the text team in Phase One follow the text recommendations of the UBS handbooks in the NT. Those recommendations usually follow the UBS text variation committee's recommendations, which I think are reasonable. Another advantage of following them is that many translators the world over, I believe, are following them, too, so it has the effect of putting everyone in the same textual ballpark, which is a good thing. I don't think it's wise to go off on our own tangent in these matters.

**Comment by TomWarren** [comment]
There are many variants of the "D" level that we have not addressed. Just going through A Textual Commentary, and I have been getting requests for footnotes. I am not sure that the suggestion has always been followed. It would be good to at least check our ULB (NT) against A Textual Commentary. I would not think that every UBS suggestion has been implemented. If it has, I would be greatly surprised.

**Comment by pohlig** [comment]
But I don't think we can load up the ULB with lots of footnotes, right?

**Comment by TomWarren** [comment]
I think the way that variants are chosen may have been more arbitrary than intentional. Just by checking the "D" readings in the A Textual Commentary, we'd know if we were on spot.

**Comment by pohlig** [comment]
D readings are inherently ambiguous. Very hard to follow any principle with
them.
Jim

**Comment by TomWarren** [comment]
My point is that if there is no checking of which were and which were not included, we have no way of being certain that the important (UBS or Textual Commentary) were included or not. My point is that we may want to check whether the most significant readings (usually the "D" readings, not that  they were the worst readings, but that they were the most difficult to make a decision about and the scholars were divided ("D" does not be worst, it means the degree of certainty is the greatest).

**Comment by TomWarren** [comment]
I think a survey of the Textual Commentary would give us a core set of the most egregious textual problems and give us a measure against what we he have in the ULB.  It could be a little project.
